models/db/score.model.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+module.exports = (sequelize, DataTypes) => {
+  const Score = sequelize.define('Score', {
+    uri: DataTypes.STRING,
+    unaware: DataTypes.INTEGER,
```

Comment:
Sequelize supports automatic validation for columns. We should use that the integer fields and confidence.
http://docs.sequelizejs.com/manual/tutorial/models-definition.html#validations